src/Executor/ManifestExecutor.php

```diff
@@ -110,6 +102,44 @@ public function runModuleManifestExecution(
         return $this->integratorLockWriter->storeLock($lockedModules);
     }
 
+    /**
+     * @param array<mixed> $unappliedManifestByType
+     * @param string $manifestType
+     * @param string $moduleName
+     * @param \SprykerSdk\Integrator\Dependency\Console\InputOutputInterface $inputOutput
+     * @param bool $isDry
+     *
+     * @return void
+     */
+    protected function applyManifestsByType(
+        array $unappliedManifestByType,
+        string $manifestType,
+        string $moduleName,
+        InputOutputInterface $inputOutput,
+        bool $isDry
+    ): void {
+        try {
+            $manifestExecutor = $this->resolveExecutor($manifestType);
+        } catch (RuntimeException $runtimeException) {
+            return;
```

Comment:
In general logic, we shouldn't have any places in the code where we catch the exception and just skip it. We should always at least log the exception somewhere or output it.

I know that this exception is possible only when we can't find the manifest type and it shouldn't happen, but it can happen, because we allow to edit the manifests manually and someone can edit the manifest type. 

So it will be good if you can add the output for this exception here as well.

Reply:
Thanks, Added warning here